Realm/RLMPlatform.h.in

```diff
@@ -16,6 +16,16 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#if REALM_BUILDING_FOR_OS_X
+
+#if TARGET_OS_IPHONE
+#error Attempting to use Realm''s OSX framework in an iOS project.
```

Comment:
Yeah. Without the second single quote `unifdef` mangles the line. `scripts/generate-rlmplatform.sh` removes the extra single quote from `unifdef`'s output.